tests/testthat/test-row-to-names.R

```diff
@@ -223,3 +248,34 @@ test_that("find_header works within row_to_names", {
     setNames(find_correct[4:nrow(find_correct),], c("D", "E"))
   )
 })
+
+test_that("multiple rows input works", {
+  q_row_to_names <- purrr::quietly(row_to_names)
+  
+  expect_equal(
+    q_row_to_names(example_data_row_to_names[[1]], row_number=1) %>% 
+      purrr::pluck("result") %>% 
+      names(),
+    c("NA", "NA")
+  )
+  
+  expect_equal(
+    q_row_to_names(example_data_row_to_names[[1]], row_number=c(1,1)) %>% 
+      purrr::pluck("result") %>% 
+      names(),
+    c("NA", "NA")
+  )
+  
+  expect_equal(
+    row_to_names(example_data_row_to_names[[1]], row_number=1:2) %>% 
+      names(),
+    c("Title", "Title2")
+  )
+  
+  expect_equal(
+    row_to_names(example_data_row_to_names[[1]], row_number=1:5) %>% 
+      names(),
+    c("Title_1_2_3", "Title2_4_5_6")
+  )
+  
+})
```

Comment:
Can you please add one more test for multiple NA values being combined and becoming a single "NA"?

Reply:
I added another test, but there was also an existing test where I tried to do this without creating another data frame:

```
expect_equal(
    suppressWarnings(
      row_to_names(example_data_row_to_names[[1]], row_number=c(1,1)) %>% 
        names()
    ),
    c("NA", "NA")
  )
```

You can either keep both tests, or remove one of them to your liking.

Reply:
Thanks, I appreciate the new more explicit test.